modules/components/src/ContextProvider/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { createContext, useContext, useState } from 'react';
```

Comment:
this file creates the context, and exports its provider and a hook to access the data in it.